### PR TITLE
whitebit.fetchDepositWtihdrawFees

### PR DESCRIPTION
**Pull request by samgermain**
2022-11-25T21:38:17.311Z
Node.js: v18.4.0
CCXT v2.2.35
whitebit.fetchDepositWithdrawFees ()
2022-11-25T21:38:21.117Z iteration 0 passed in 1122 ms
```js
{
  '1INCH': {
    info: {
      '1INCH': {
        is_depositable: true,
        is_withdrawal: true,
        ticker: '1INCH',
        name: '1inch',
        providers: [],
        withdraw: {
          max_amount: '0',
          min_amount: '25',
          fixed: '3.07',
          flex: null
        },
        deposit: { max_amount: '0', min_amount: '15', fixed: null, flex: null }
      }
    },
    withdraw: { fee: 3.07, percentage: false },
    deposit: { fee: undefined, percentage: undefined },
    networks: {}
  },
  ...
  ZRX: {
    info: {
      ZRX: {
        is_depositable: true,
        is_withdrawal: true,
        ticker: 'ZRX',
        name: '0x',
        providers: [],
        withdraw: {
          max_amount: '0',
          min_amount: '55',
          fixed: '8.26',
          flex: null
        },
        deposit: { max_amount: '0', min_amount: '45', fixed: null, flex: null }
      }
    },
    withdraw: { fee: 8.26, percentage: false },
    deposit: { fee: undefined, percentage: undefined },
    networks: {}
  }
}
```
2022-11-25T21:38:21.117Z iteration 1 passed in 1122 ms

-------------------------

2022-11-25T21:43:11.975Z
Node.js: v18.4.0
CCXT v2.2.35
whitebit.fetchDepositWithdrawFees (BTC,USDT)
2022-11-25T21:43:16.214Z iteration 0 passed in 1032 ms

```js
{
  BTC: {
    info: {
      BTC: {
        is_depositable: true,
        is_withdrawal: true,
        ticker: 'BTC',
        name: 'Bitcoin',
        providers: [],
        withdraw: {
          max_amount: '0',
          min_amount: '0.001',
          fixed: '0.0004',
          flex: null
        },
        deposit: {
          max_amount: '0',
          min_amount: '0.0005',
          fixed: null,
          flex: null
        }
      }
    },
    withdraw: { fee: 0.0004, percentage: false },
    deposit: { fee: undefined, percentage: undefined },
    networks: {}
  },
  USDT: {
    info: {
      'USDT (BEP20)': {
        is_depositable: true,
        is_withdrawal: true,
        ticker: 'USDT',
        name: 'Tether US',
        providers: [],
        withdraw: { max_amount: '0', min_amount: '10', fixed: '1', flex: null },
        deposit: { max_amount: '0', min_amount: '5', fixed: null, flex: null }
      },
      'USDT (EOS)': {
        is_depositable: true,
        is_withdrawal: true,
        ticker: 'USDT',
        name: 'Tether US',
        providers: [],
        withdraw: { max_amount: '0', min_amount: '15', fixed: '5', flex: null },
        deposit: { max_amount: '0', min_amount: '10', fixed: null, flex: null }
      },
      'USDT (ERC20)': {
        is_depositable: true,
        is_withdrawal: true,
        ticker: 'USDT',
        name: 'Tether US',
        providers: [],
        withdraw: {
          max_amount: '0',
          min_amount: '25',
          fixed: '2.06',
          flex: null
        },
        deposit: { max_amount: '0', min_amount: '15', fixed: null, flex: null }
      },
      'USDT (OMNI)': {
        is_depositable: true,
        is_withdrawal: true,
        ticker: 'USDT',
        name: 'Tether US',
        providers: [],
        withdraw: { max_amount: '0', min_amount: '30', fixed: '15', flex: null },
        deposit: { max_amount: '0', min_amount: '60', fixed: '15', flex: null }
      },
      'USDT (TRC20)': {
        is_depositable: true,
        is_withdrawal: true,
        ticker: 'USDT',
        name: 'Tether US',
        providers: [],
        withdraw: { max_amount: '0', min_amount: '10', fixed: '1', flex: null },
        deposit: { max_amount: '0', min_amount: '5', fixed: null, flex: null }
      }
    },
    withdraw: { fee: undefined, percentage: undefined },
    deposit: { fee: undefined, percentage: undefined },
    networks: {
      BEP20: {
        withdraw: { fee: 1, percentage: false },
        deposit: { fee: undefined, percentage: undefined }
      },
      EOS: {
        withdraw: { fee: 5, percentage: false },
        deposit: { fee: undefined, percentage: undefined }
      },
      ERC20: {
        withdraw: { fee: 2.06, percentage: false },
        deposit: { fee: undefined, percentage: undefined }
      },
      OMNI: {
        withdraw: { fee: 15, percentage: false },
        deposit: { fee: 15, percentage: false }
      },
      TRC20: {
        withdraw: { fee: 1, percentage: false },
        deposit: { fee: undefined, percentage: undefined }
      }
    }
  }
}
```
2022-11-25T21:43:16.214Z iteration 1 passed in 1032 ms

----------------

2022-11-25T21:44:44.485Z
Node.js: v18.4.0
CCXT v2.2.35
whitebit.fetchDepositWithdrawFee (USDT)
2022-11-25T21:44:48.169Z iteration 0 passed in 1053 ms

```js
{
  info: {
    'USDT (BEP20)': {
      is_depositable: true,
      is_withdrawal: true,
      ticker: 'USDT',
      name: 'Tether US',
      providers: [],
      withdraw: { max_amount: '0', min_amount: '10', fixed: '1', flex: null },
      deposit: { max_amount: '0', min_amount: '5', fixed: null, flex: null }
    },
    'USDT (EOS)': {
      is_depositable: true,
      is_withdrawal: true,
      ticker: 'USDT',
      name: 'Tether US',
      providers: [],
      withdraw: { max_amount: '0', min_amount: '15', fixed: '5', flex: null },
      deposit: { max_amount: '0', min_amount: '10', fixed: null, flex: null }
    },
    'USDT (ERC20)': {
      is_depositable: true,
      is_withdrawal: true,
      ticker: 'USDT',
      name: 'Tether US',
      providers: [],
      withdraw: { max_amount: '0', min_amount: '25', fixed: '2.27', flex: null },
      deposit: { max_amount: '0', min_amount: '15', fixed: null, flex: null }
    },
    'USDT (OMNI)': {
      is_depositable: true,
      is_withdrawal: true,
      ticker: 'USDT',
      name: 'Tether US',
      providers: [],
      withdraw: { max_amount: '0', min_amount: '30', fixed: '15', flex: null },
      deposit: { max_amount: '0', min_amount: '60', fixed: '15', flex: null }
    },
    'USDT (TRC20)': {
      is_depositable: true,
      is_withdrawal: true,
      ticker: 'USDT',
      name: 'Tether US',
      providers: [],
      withdraw: { max_amount: '0', min_amount: '10', fixed: '1', flex: null },
      deposit: { max_amount: '0', min_amount: '5', fixed: null, flex: null }
    }
  },
  withdraw: { fee: undefined, percentage: undefined },
  deposit: { fee: undefined, percentage: undefined },
  networks: {
    BEP20: {
      withdraw: { fee: 1, percentage: false },
      deposit: { fee: undefined, percentage: undefined }
    },
    EOS: {
      withdraw: { fee: 5, percentage: false },
      deposit: { fee: undefined, percentage: undefined }
    },
    ERC20: {
      withdraw: { fee: 2.27, percentage: false },
      deposit: { fee: undefined, percentage: undefined }
    },
    OMNI: {
      withdraw: { fee: 15, percentage: false },
      deposit: { fee: 15, percentage: false }
    },
    TRC20: {
      withdraw: { fee: 1, percentage: false },
      deposit: { fee: undefined, percentage: undefined }
    }
  }
}
```
2022-11-25T21:44:48.169Z iteration 1 passed in 1053 ms

### CODE REVIEW

File: js/whitebit.js [hunk 1]
```diff
@@ -221,6 +223,11 @@ module.exports = class whitebit extends Exchange {
                     'margin': 'collateral',
                     'trade': 'spot',
                 },
+                'networksById': {
+                    'BEP20': 'BSC',
+                    'ERC20': 'ETH',
```
Comment (carlosmiei) [inline js/whitebit.js:228]: @samgermain again please do not add these entries (trc20, erc20) here

**Comment by carlosmiei** [comment]
LGTM ✅ 

```
p whitebit fetchDepositWithdrawFees '["USDT"]'
Python v3.10.8
CCXT v2.2.103
whitebit.fetchDepositWithdrawFees(['USDT'])
{'USDT': {'deposit': {'fee': None, 'percentage': None},
          'info': {'USDT (BEP20)': {'deposit': {'fixed': None,
                                                'flex': None,
                                                'max_amount': '0',
                                                'min_amount': '5'},
                                    'is_depositable': True,
                                    'is_withdrawal': True,
                                    'name': 'Tether US',
                                    'providers': [],
                                    'ticker': 'USDT',
                                    'withdraw': {'fixed': '1',
                                                 'flex': None,
                                                 'max_amount': '0',
                                                 'min_amount': '10'}},
                   'USDT (EOS)': {'deposit': {'fixed': None,
                                              'flex': None,
                                              'max_amount': '0',
                                              'min_amount': '10'},
                                  'is_depositable': True,
                                  'is_withdrawal': True,
                                  'name': 'Tether US',
                                  'providers': [],
                                  'ticker': 'USDT',
                                  'withdraw': {'fixed': '5',
                                               'flex': None,
                                               'max_amount': '0',
                                               'min_amount': '15'}},
                   'USDT (ERC20)': {'deposit': {'fixed': None,
                                                'flex': None,
                                                'max_amount': '0',
                                                'min_amount': '15'},
                                    'is_depositable': True,
                                    'is_withdrawal': True,
                                    'name': 'Tether US',
                                    'providers': [],
                                    'ticker': 'USDT',
                                    'withdraw': {'fixed': '5.61',
                                                 'flex': None,
                                                 'max_amount': '0',
                                                 'min_amount': '25'}},
                   'USDT (OMNI)': {'deposit': {'fixed': '15',
                                               'flex': None,
                                               'max_amount': '0',
                                               'min_amount': '60'},
                                   'is_depositable': True,
                                   'is_withdrawal': True,
                                   'name': 'Tether US',
                                   'providers': [],
                                   'ticker': 'USDT',
                                   'withdraw': {'fixed': '15',
                                                'flex': None,
                                                'max_amount': '0',
                                                'min_amount': '30'}},
                   'USDT (TRC20)': {'deposit': {'fixed': None,
                                                'flex': None,
                                                'max_amount': '0',
                                                'min_amount': '5'},
                                    'is_depositable': True,
                                    'is_withdrawal': True,
                                    'name': 'Tether US',
                                    'providers': [],
                                    'ticker': 'USDT',
                                    'withdraw': {'fixed': '1',
                                                 'flex': None,
                                                 'max_amount': '0',
                                                 'min_amount': '10'}}},
          'networks': {'BSC': {'deposit': {'fee': None, 'percentage': None},
                               'withdraw': {'fee': 1.0, 'percentage': False}},
                       'EOS': {'deposit': {'fee': None, 'percentage': None},
                               'withdraw': {'fee': 5.0, 'percentage': False}},
                       'ERC20': {'deposit': {'fee': None, 'percentage': None},
                                 'withdraw': {'fee': 5.61,
                                              'percentage': False}},
                       'OMNI': {'deposit': {'fee': 15.0, 'percentage': False},
                                'withdraw': {'fee': 15.0, 'percentage': False}},
                       'TRC20': {'deposit': {'fee': None, 'percentage': None},
                                 'withdraw': {'fee': 1.0,
                                              'percentage': False}}},
          'withdraw': {'fee': None, 'percentage': None}}}
```

File: js/whitebit.js (hunk 1 above)
Comment (samgermain) [reply js/whitebit.js]: but we're keeping `BEP20 -> BSC` ?

Comment (carlosmiei) [reply js/whitebit.js]: @samgermain tbh I'm sure what's the "correct" value for the BSC, currently, we have different notations (BSC/BEP20) across the board but yeah, for the sake of consistency we should probably remove them and add this entry to `defaultNetworkCodeReplacements`, wdyt? 
```
            'defaultNetworkCodeReplacements': {
                'BSC': { 'BEP20': 'BNB' },
```

Comment (samgermain) [reply js/whitebit.js]: BNB corresponds to BEP2
BSC corresponds to BEP20

**Pull request merged**